warehouse/query-core/src/main/java/datawave/query/dashboard/DashboardQueryLogic.java

```diff
@@ -20,6 +20,8 @@
  */
 public class DashboardQueryLogic extends ShardQueryLogic implements QueryLogicTransformer {
     
+    protected long queryExecutionForCurrentPageStartTime;
```

Comment:
I don't think this is needed here.

Reply:
this class implements QueryLogicTransformer, which  now has `void setQueryExecutionForPageStartTime(long queryExecutionForCurrentPageStartTime);`, so this variable is used in the implementation of that method.

Reply:
oh right, of course.  I missed that.